### PR TITLE
fix(progress): improve description/status text/wrapping

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/3002
fixes https://github.com/patternfly/patternfly/issues/3434

aligns the description/status text to the bottom, adds `word-break`, right aligns status text, improves behavior when both description and status have a lot of content.

### CODE REVIEW

File: src/patternfly/components/Progress/progress.scss [hunk 1]
```diff
@@ -109,11 +109,15 @@
     }
   }
 
+  &.pf-m-outside,
+  &.pf-m-singleline {
+    grid-template-columns: 1fr fit-content(50%);
```
Comment (mattnolting) [inline src/patternfly/components/Progress/progress.scss:114]: 👍

**Review by mattnolting — APPROVED** [review]
Great work, LPTM 👍

File: src/patternfly/components/Progress/progress.scss [hunk 2]
```diff
@@ -19,7 +19,6 @@
   --pf-c-progress--m-success__status-icon--Color: var(--pf-global--success-color--100);
   --pf-c-progress--m-inside__indicator--MinWidth: var(--pf-global--spacer--xl);
   --pf-c-progress--m-inside__measure--Color: var(--pf-global--Color--light-100); // the measure needs to be white when moved inside the progress bar indicator
-  --pf-c-progress--m-success--m-inside__measure--Color: var(--pf-global--Color--dark-100); // the measure still needs to be dark when in success state
```
Comment (christiemolloy) [inline src/patternfly/components/Progress/progress.scss:22]: breaking change ?

File: src/patternfly/components/Progress/progress.scss [hunk 3]
```diff
@@ -126,14 +130,16 @@
 
 // the progress__description is displayed above the progress bar on the left
 .pf-c-progress__description {
+  word-break: break-word;
```
Comment (christiemolloy) [inline src/patternfly/components/Progress/progress.scss:133]: should we have an option to truncate it ? @mcarrano

**Review by christiemolloy — CHANGES_REQUESTED** [review]

File: src/patternfly/components/Progress/progress.scss (hunk 2 above)
Comment (mcoker) [reply src/patternfly/components/Progress/progress.scss:22]: oh good catch! added back.

File: src/patternfly/components/Progress/progress.scss (hunk 3 above)
Comment (mcoker) [reply src/patternfly/components/Progress/progress.scss:135]: ping @mcarrano @mceledonia

Comment (mcarrano) [reply src/patternfly/components/Progress/progress.scss:135]: Yes, I think it would make sense to offer that option, if possible.

Comment (mcoker) [reply src/patternfly/components/Progress/progress.scss:144]: @christiemolloy @mcarrano added truncation

**Review by mcarrano — APPROVED** [review]
Looks good @mcoker

**Review by christiemolloy — APPROVED** [review]
LGTM!

**Pull request merged**